### ISSUE TITLE
Update Students' details in batches

### ISSUE THREAD

**Issue by joshuayapwj98** [AY2122S2-CS2103T-T12-1/tp#52]
When a student is COVID-19 positive, there should be a straightforward process (method) to batch update students related to the COVID-19 positive student.

Group Conversation: 
- Change JSON format to categorize Students 
- Array of Class Codes which stores an Array of Persons (Another implementation of JSON Address Book)

**Comment by joshuayapwj98** [comment]
To discuss: What if the student who was COVID-19 positive changed their status back to Negative, what should be the follow-up action?

During our team discussion, we could not derive a standard solution. However, some suggestions include:
- Utilize the same method call to return close contact status to Negative. This approach seems viable but might have confused __edge cases__.
- Let the change of status be independent. In addition, add a __timer__ element to the status (epoch library?).

In conclusion, the team will further discuss future iterations to implement this feature.
